app/assets/stylesheets/AdminLTE.scss

```diff
@@ -2669,6 +2669,7 @@ a:focus {
 .table-bordered > tbody > tr > td,
 .table-bordered > tfoot > tr > td {
   border: 1px solid $diaper-color-neutral-96;
+  font-size: 17px;
```

Comment:
I'm not seeing any `tfoot` tags in the `:show` view that you edited -- is this related to the change?

Reply:
No tfoot is not related to my code, i added font-size in that so it look better in table-bordered class.
But i realised by doing this it might effect other places where table-bordered classes right?

Reply:
@armahillo I updated the code can you please review this.

Reply:
Thanks!

Generally speaking, let's keep the changes in-scope to the issue and not add stuff before we need to, to avoid side effects and unintended consequences :)

Reply:
Just checked:
```
aaron@toph:~/Documents/Code/rails/diaper$ rg table-bordered app/views | wc -l
0
aaron@toph:~/Documents/Code/rails/diaper$ rg tfoot app/views | wc -l
6
```
We have 3 tfoots (`StorageLocations#index`) but no `table-bordered` so this change is harmless for now. :+1: